TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/transform/SigV4AuthExcisionTest.java

```diff
@@ -0,0 +1,58 @@
+package org.opensearch.migrations.transform;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.CharStreams;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.opensearch.migrations.replay.datahandlers.JsonAccumulator;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.LinkedHashMap;
+
+public class SigV4AuthExcisionTest {
+
+    static ObjectMapper objectMapper = new ObjectMapper();
+
+
+    static InputStream getInputStreamForTypeMappingResource(String resourceName) {
```

Comment:
Please rename (still TypeMapping from copy-paste).
Also, please adjust the extensions to be .json instead of .txt